website/docs/reference/warehouse-profiles/iomete-profile.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Apache iomete Profile"
+---
+
+## Overview of dbt-iomete
+
+**Maintained by:** iomete
+**Author:** Namig Aliyev
+**Source:** [Github](https://github.com/iomete/dbt-iomete)
+**dbt Cloud:** Currently un-supported
+**dbt Slack channel** [Link to channel]()
```

Comment:
if you want a Slack channel, please request one in the [community Slack](https://www.getdbt.com/community/join-the-community/)'s [#channel-requests channel](https://getdbt.slack.com/archives/C01D8J8AJDA)!

Reply:
I already mentioned about to create a dedicated channel for dbt-iomete in #channel-request.